pkg/commands/entrypoints/entrypoints.go

```diff
@@ -130,7 +131,18 @@ func (c *EntrypointsCommand) Flags() *cli.FlagSet {
 		Default: false,
 	})
 
+	f.BoolVar(&cli.BoolVar{
+		Name:    "no-recursion",
```

Comment:
Not sure im a fan of the naming, `skip-nested-entrypoints`? Something with skip to keep naming similar.

Reply:
Done